backend/mariadb_backend.ml

```diff
@@ -2,6 +2,11 @@ open CCFun
 open Lwt.Infix
 open Caqti_request.Infix
 
+let combine_lwt m =
```

Comment:
Couldn't that be replaced by an `Lwt.bind` ?

Reply:
No, I was looking for a function with signature `'a Lwt.t Lwt.t -> 'a Lwt.t` and this was the quickest solution ;-)